CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+### PaymentSheet, CustomerSheet
+* [Changed] Changed the edit and remove saved payment method flow so that tapping 'Edit' displays an icon that leads to a new update payment method screen
```

Comment:
Nit:
- We're missing a new line under this
- Need a period to end the sentence (see other entries)
- We should explain what this new payment method screen does e.g. displays card cvc, expiry, bank acct details, sepa, etc.